omg/entities/projectile.py

```diff
@@ -133,57 +76,24 @@ def create(cls, init_px, init_py, angle) -> Projectile:
         )
 
 
-class FireIceFactory(ProjectileFactory):
-    """Convenience class to create IceShard Projectile."""
-
-    image_file = os.path.join(ASSET_DIR, "skills", "FireIce.PNG")
-    scale = 0.05
-    damage = 15
-    speed = 7
-    mana_cost = 20
-
-    @classmethod
-    def create(cls, init_px, init_py, angle) -> Projectile:
-        """Create an IceShard projectile."""
-        return Projectile(
-            name=cls.__name__,
-            image_file=cls.image_file,
-            scale=cls.scale,
-            damage=cls.damage,
-            init_px=init_px,
-            init_py=init_py,
-            speed=cls.speed,
-            angle=angle,
-        )
-
+with open(CRAFTED_SKILLS_JSON_DIR, "r") as file:
+    crafted_skill_dictionary: Dict[str, Dict] = json.load(file)
 
-class FireFireFactory(ProjectileFactory):
-    """Convenience class to create IceShard Projectile."""
 
-    image_file = os.path.join(ASSET_DIR, "skills", "FireFire.PNG")
-    scale = 0.05
-    damage = 15
-    speed = 7
-    mana_cost = 20
+class CraftingSkillFactory(ProjectileFactory):
```

Comment:
if this is a `factory` design creating a `Skill`, Can I suggest using `SkillFactory` as a name?